extension/model/modelRegistry.js

```diff
@@ -215,24 +215,24 @@ const modelRegistry = {
   ende: {
     vocab: {
       name: "vocab.deen.spm",
```

Comment:
Can I convince you for duplication here of the same vocab as `vocab.ende.spm` with an argument of consistency for easy automation in https://github.com/browsermt/bergamot-translator/pull/319?

Reply:
```
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/bergamot-models-sandbox/0.2.10/ende/vocab.ende.spm
```

Reply:
Creating `vocab.deen.spm` as `vocab.ende.spm` creates a false sense that the two vocab files are different while for most of the language pairs the vocab file for the pair EN <-> Non-EN is just 1 file (e.g. See [this](https://github.com/browsermt/students/blob/master/deen/download-models.sh#L8) and [this](https://github.com/browsermt/students/blob/master/deen/download-models.sh#L22)).

This also keeps the name consistent with the [students](https://github.com/browsermt/students) repository. We took this conscious decision way back when we set up model repository within mozilla org.

Reply:
I'm going to go ahead with something like https://gist.github.com/jerinphilip/f9bc31d70e4201771b7734ab1cfd1205 when it gets to working eventually. Ideally, this should come from Mozilla, who maintains a separate model inventory. If you want your models tested (by means of automation provisioned from elsewhere) you are welcome to provide the equivalent and also use the same JSON in the extension than complain when multiple manual copy-pastes break model and configuration. 

I will update the URL to a link provided by you with the equivalent JSON if that ever happens.

Reply:
Marking as resolved.